### ISSUE TITLE
Audiobookshelf bundle focus styles

### ISSUE THREAD

**Issue by axelnanol** [axelnanol/tizenportal#41]
**Describe the bug**
The abs bundle doesn't have styles round its focus elements. You have to turn focus outlines on to see where you are, or watch the transitions.

**To Reproduce**
1. Set up an Audiobookshelf server and add as site card
2. Launch
3. Note the lack of focus ring around your current focus element.

**Expected behavior**
Consistent behaviour for custom focus elements and automatic ones

**Additional context**
We should probably merge automatic and registered elements into one list. Perhaps we can even analyse how abs elements are being selected (across all page types), and any that are either already included in automatic selection, or it makes sense to move to there, can be moved.
